packages/admin-ui/src/components/05_pages/NodeForm/index.js

```diff
@@ -49,13 +49,21 @@ class NodeForm extends React.Component {
     setErrorMessage: PropTypes.func.isRequired,
     onChange: PropTypes.func,
     requestComponentList: PropTypes.func.isRequired,
+    components: PropTypes.shape({
```

Comment:
The error is probably on the area of widgets, which weren't yet documented as proptypes.